commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -45,8 +45,12 @@
         <section class="hosted-tone--dark">
             <div class="host hosted__container--full">
                 <div class="u-responsive-ratio u-responsive-ratio--hd">
-                    @if(page.video.youTubeHtml.isDefined){
-                        @Html(page.video.youTubeHtml.get)
+                    @if(page.video.youtubeId.isDefined){
+                        <iframe id="hosted-video-@{page.video.youtubeId}"
+                        class="js-hosted-youtube-video"
+                        src="https://www.youtube.com/embed/@{page.video.youtubeId}?modestbranding=1&showinfo=0&rel=0&enablejsapi=1"
+                        frameborder="0"
+                        allowfullscreen=""></iframe>
                     } else {
```

Comment:
Probably want:
```
@page.video.youtubeId.map { youtubeId =>
...
} getOrElse {
...
```
otherwise iframe id will be something like `hosted-video-Some(id)`